src/stavetext.ts

```diff
@@ -1,25 +1,33 @@
 // [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
 // Author Taehoon Moon 2014
 
+import { FontInfo, FontStyle, FontWeight } from './font';
 import { Stave } from './stave';
 import { StaveModifier, StaveModifierPosition } from './stavemodifier';
 import { Justification, TextNote } from './textnote';
-import { FontInfo } from './types/common';
 import { RuntimeError } from './util';
 
 export class StaveText extends StaveModifier {
   static get CATEGORY(): string {
     return 'StaveText';
   }
 
-  protected text: string;
-  protected shift_x?: number;
-  protected shift_y?: number;
+  static TEXT_FONT: Required<FontInfo> = {
+    family: 'Times' /* RONYEH: Font.SERIF*/,
+    size: 16,
+    weight: FontWeight.NORMAL,
+    style: FontStyle.NORMAL,
+  };
+
   protected options: {
     shift_x: number;
     shift_y: number;
     justification: number;
   };
+
+  protected text: string;
+  protected shift_x?: number;
+  protected shift_y?: number;
```

Comment:
If those variables are internal and not accessed from outside (seems to be the case) Consider using cammelCase

Reply:
The constructor takes an options param of the type `options: { shift_x?: number; shift_y?: number; justification?: number }`. Weirdly enough, this class has a shift_x and also a options.shift_x stored internally. Are they redundant? Who knows? Let's revisit this after releasing 4.0. I think folks are probably anxious to get to beta soon. :-)